Makefile

```diff
@@ -488,7 +488,7 @@ docs/spec/%: bundle/community/manifests/% | gen-api-docs
 	$(GEN_API_DOCS) < $^ > $@
 
 docs/operator/config.yaml: gen-api-docs
-	$(GEN_API_DOCS) -pkg github.com/grafana/tempo-operator/apis/config/v1alpha1 -type ProjectConfig -format multiline > $@
+	$(GEN_API_DOCS) -pkg github.com/grafana/tempo-operator/api/config/v1alpha1 -type ProjectConfig -format multiline > $@
```

Comment:
Can you explain this change? Maybe a link to documentation? 

I think the `apis` was used for multi version projects.

Reply:
and our PROJECT uses multigroup

Reply:
Not anymore. is part of the layout changes

[New layout:](https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4#new-layout)

    The directory apis was renamed to api to follow the standard
    The controller(s) directory has been moved under a new directory called internal and renamed to singular as well controller
    The main.go previously scaffolded in the root directory has been moved under a new directory called cmd